### ISSUE TITLE
Add section 10 minutes to FugueSQL

### ISSUE THREAD

**Issue by rdmolony** [fugue-project/tutorials#106]
Leaving the remaining sections for a deeper dive

**Comment by rdmolony** [comment]
On second thoughts, this is already well covered by `syntax`, additional sections should be added there

**Comment by kvnkho** [comment]
Yes this one would be good for sure. There is too much theory included in the current FugueSQL introduction.
